authors/apps/authentication/models.py

```diff
@@ -111,3 +116,28 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
+
+    @property
+    def token(self):
+        """
+        We need to make the method for creating our token private. At the
+        same time, it's more convenient for us to access our token with
+        `user.token` and so we make the token a dynamic property by wrapping
+        in in the `@property` decorator.
+        """
+        return self._generate_jwt_token()
+
+    def _generate_jwt_token(self):
+        """
+        We generate JWT token and add the user id, username and expiration
+        as an integer.
+        """
+        dt = datetime.now() + timedelta(hours=2)
```

Comment:
@MandelaK, please give the `dt` variable a more descriptive name.

Reply:
Done